desktop/windows/release-notes/index.md

```diff
@@ -52,6 +52,7 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 ### Security
 
 - Fixed [CVE-2022-0847](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-0847){: target="_blank" rel="noopener" class="_"}, aka “Dirty Pipe”, an issue that could enable attackers to modify files in container images on the host, from inside a container.
+  If using the WSL 2 backend, you must [update WSL 2 from Microsoft](https://www.catalog.update.microsoft.com/Search.aspx?q=wsl) to a version equal or greater than 5.10.102.
```

Comment:
There is the `wsl --update` command to do that more easily

Reply:
Good point, updated!